app/src/main/java/com/duckduckgo/app/tabs/ui/TabSwitcherActivity.kt

```diff
@@ -167,6 +183,20 @@ class TabSwitcherActivity : DuckDuckGoActivity(), TabSwitcherListener, Coroutine
                 onDeletableTab(it.last())
             }
         }
+
+        lifecycleScope.launch {
+            viewModel.isFeatureAnnouncementVisible
+                .flowWithLifecycle(lifecycle, Lifecycle.State.STARTED)
+                .collect { isVisible ->
+                    if (isVisible) {
+                        viewModel.onTabFeatureAnnouncementDisplayed()
+                        announcement.visibility = View.VISIBLE
```

Comment:
You can use `announcement.show()` / `announcement.gone()`.

Reply:
Done in [6054d81](https://github.com/duckduckgo/Android/pull/4743/commits/6054d81879b1a47d62ea0bb465ff877e474068a6).